src/main/java/com/xceptance/xlt/report/util/MovingArerage.java

```diff
@@ -0,0 +1,134 @@
+/*
+ * Copyright (c) 2005-2022 Xceptance Software Technologies GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.xceptance.xlt.report.util;
+
+/**
+ * Hold the data for additional moving averages.
+ */
+public class MovingArerage
+{
+    /**
+     * Additional moving averages. 
+     */
+    public enum MovingAverages
```

Comment:
What about to rename this enum to something that better fits its intention, e.g. `MovingAverageType`?